tests/keras/layers/test_advanced_activations.py

```diff
@@ -0,0 +1,139 @@
+import unittest
+from numpy.testing import assert_allclose
+import numpy as np
+from keras import backend as K
+
+np.random.seed(1337)
+
+def get_standard_values():
+    '''
+    These are just a set of floats used for testing the activation
+    functions, and are useful in multiple tests.
+
+    The values should all be non-negative becasue they and their negative
+    are used to test the ReLU derivates in this file.
+    '''
+    return np.array([[0, 0.1, 0.5, 0.9, 1.0, 10, 1e3, 0.0001]], dtype=K.floatx())
+
+class TestAdvancedActivations(unittest.TestCase):
```

Comment:
there are several ways you can run tests standalone with pytest.
 - pytest.main(...)
 - from terminal: py.test -k test_name_or_file_name_or_class_or_prefix

I usually just run them from main as a standalone function. 

pytest is way superior than unittest, so I strongly suggest to just switch once and for all:)